libpod/container.go

```diff
@@ -553,6 +553,13 @@ func (c *Container) Init() (err error) {
 	if err := c.mountStorage(); err != nil {
 		return err
 	}
+	defer func() {
+		if err != nil {
+			if err2 := c.cleanupStorage(); err2 != nil {
+				logrus.Errorf("Error tearing down network namespace for container %s: %v", c.ID(), err2)
```

Comment:
Error message is wrong, this is tearing down storage not the network

Reply:
Fixed